### PR TITLE
api-controller bump & api-controller-acceptance-tests bump

### PR DESCRIPTION
**Pull request by strekm**
**Description**

Changes proposed in this pull request:

- api-controller bump

**Related issue(s)**
see #2694

### CODE REVIEW

**Comment by kubadz** [comment]
I tested this manually with some additional cases not covered in tests, and everything works fine. 

I would give an approve but I have one question: 
are you planning to set blacklisted service mentioned in issue (`kubernetes.default`) in a proper ENV in api-controller deployment as a part of this PR, or you will create another PR with that change?

File: resources/core/charts/api-controller/values.yaml [hunk 1]
```diff
@@ -9,4 +9,6 @@ mtls:
 cors:
   allow_origin: "*"
   allow_methods: "GET,POST,PUT,DELETE" # no possibility to use "*"
-  allow_headers: "*"
\ No newline at end of file
+  allow_headers: "*"
+
+blacklisted_services: "kubernetes.default,istio-citadel.istio-system,istio-galley.istio-system,istio-ingressgateway.istio-system,istio-pilot.istio-system,istio-policy.istio-system,istio-sidecar-injector.istio-system,istio-telemetry.istio-system,apiserver-proxy.kyma-system"
```
Comment (piotrmsc) [inline resources/core/charts/api-controller/values.yaml]: There is also apiserver-proxy-ssl

**Pull request merged**